README.md

```diff
@@ -1,16 +1,50 @@
-# ![nfcore/test-datasets](docs/images/test-datasets_logo.png)
-Test data to be used for automated testing with the nf-core pipelines
+# Test data sets: `pangenome`
 
-## Introduction
+This branch contains test data to be used for automated testing with the nf-core/pangenome pipeline.
 
-nf-core is a collection of high quality Nextflow pipelines.
+## Content of this repository
 
-## Documentation
-nf-core/test-datasets comes with documentation in the `docs/` directory:
+`data/`: FASTA files for minimal test data set.
 
-01. [Add a new  test dataset](https://github.com/nf-core/test-datasets/blob/master/docs/ADD_NEW_DATA.md)
-02. [Use an existing test dataset](https://github.com/nf-core/test-datasets/blob/master/docs/USE_EXISTING_DATA.md)
+## Minimal test data set origin
 
-## Support
+_DRB1-3123.fa.gz_ FASTA data set consists of twelve HLA DRB1-3123 haplotypes taking from [`pggb`](https://github.com/pangenome/pggb)'s example [data](https://github.com/pangenome/pggb/blob/master/data/HLA/DRB1-3123.fa.gz).
 
-For further information or help, don't hesitate to get in touch on our [Slack organisation](https://nf-co.re/join/slack) (a tool for instant messaging).
+### Sequence information
+
+| Sequence   | Lenght    |
+|-------------|------------|
+| gi\|568815592:32578768-32589835 | 11068 |
+| gi\|568815529:3998044-4011446 | 13403 |
+| gi\|568815551:3814534-3830133 | 15600 |
+| gi\|568815561:3988942-4004531 | 15590 |
+| gi\|568815567:3779003-3792415 | 13413 |
+| gi\|568815569:3979127-3993865 | 14739 |
+| gi\|345525392:5000-18402 | 13403 |
+| gi\|29124352:124254-137656 | 13403 |
+| gi\|28212469:126036-137103 | 11068 |
+| gi\|28212470:131613-146345 | 14733 |
+| gi\|528476637:32549024-32560088 | 11065 |
+gi\|157702218:147985-163915 | 15931 |
+
+### Example output
+
+```sh
+pggb -i /data/HLA/DRB1-3123.fa.gz -s 3000 -K 11 -p 70 -a 70 -n 10 -t 2 -v -l
+```
+
+#### Seqwish graph statistics
+| Lenght | Nodes | Edges | Paths |
+| ----- | ----- | ----- | -----|
+| 23622 | 3608 | 4913 | 12 |
+
+>Running the pipeline with the _DRB1-3123.fa.gz_ test data might result in slightly different outputs compared to what you see here. This is due to some _stochastic processes_ in [`smoothxg`](https://github.com/pangenome/smoothxg).
```

Comment:
Not a comment about this README per se, but can `smoothxg` be run with a known random number seed so that runs are deterministic, at least for testing purposes?

Reply:
```suggestion
> Running the pipeline with the _DRB1-3123.fa.gz_ test data might result in slightly different outputs compared to what you see here. This is due to some _stochastic processes_ in [`smoothxg`](https://github.com/pangenome/smoothxg).
```

Reply:
@heuermh 
There was a deterministic implementation, but somehow it got lost over the development. and it could slow down things significantly.
 @AndreaGuarracino and I are unhappy, that we don't have a deterministic version at the moment. We understand that this is not ideal. We will discuss this issue tomorrow at our hackathon meeting.

Reply:
We use [Hogwild!](https://people.eecs.berkeley.edu/~brecht/papers/hogwildTR.pdf) for our path-guided Stochastic Gradient Descent (P-SGD) approach, that's where the non-determinism comes from.